esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -28,7 +37,17 @@ def _cube_to_MeshInfo(cube):
 def _cube_to_GridInfo(cube):
     # Returns a GridInfo object describing the horizontal grid of the cube.
     # This may be inherited from code written for the rectilinear regridding scheme.
```

Comment:
We should note that we anticipate this function mirroring an equivalent function (likely with a different name) in #26.